pkg/components/mysql/mysql.go

```diff
@@ -36,6 +39,21 @@ func Cluster(mattermost *mattermostv1alpha1.ClusterInstallation) *mysqlOperator.
 		Spec: mysqlOperator.ClusterSpec{
 			Version: "8.0.12",
 			Members: 1,
+			VolumeClaimTemplate: &corev1.PersistentVolumeClaim{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: mysqlName,
+				},
+				Spec: corev1.PersistentVolumeClaimSpec{
+					AccessModes: []corev1.PersistentVolumeAccessMode{
+						"ReadWriteOnce",
+					},
+					Resources: corev1.ResourceRequirements{
+						Requests: corev1.ResourceList{
+							corev1.ResourceStorage: resource.MustParse(mattermost.Spec.DatabaseType.DbStorageSize),
```

Comment:
What happens if I enter a bad value for `DbStorageSize`? Say "abc"?

Reply:
We need to add some validation in the yaml and in the code as well. will check

Reply:
the code has this check: `resource.MustParse(mattermost.Spec.DatabaseType.DbStorageSize),` if that fail the resource will not be created. maybe we can check that in the beginning and fail @jwilander

Reply:
we can add validation in the future, in 1.14 support some validation, i need to investigate how this works.
can be in a follow up?

Reply:
add a validation, if those fields are wrong it will complain

```
The ClusterInstallation "test" is invalid: []: Invalid value: map[string]interface {}{"metadata":map[string]interface {}{"uid":"104050fe-860e-11e9-8391-2638c91ce35c", "name":"test", "namespace":"default", "creationTimestamp":"2019-06-03T14:44:17Z", "annotations":map[strin
g]interface {}{"kubectl.kubernetes.io/last-applied-configuration":"{\"apiVersion\":\"mattermost.com/v1alpha1\",\"kind\":\"ClusterInstallation\",\"metadata\":{\"annotations\":{},\"name\":\"test\",\"namespace\":\"default\"},\"spec\":{\"databaseType\":{\"dbStorageSize\":\"11
GRT\"},\"ingressName\":\"demo.cpanato.spinmint.com\",\"minioStorageSize\":\"1Gi\",\"replicas\":1}}\n"}, "generation":1}, "spec":map[string]interface {}{"replicas":1, "databaseType":map[string]interface {}{"dbStorageSize":"11GRT"}, "ingressName":"demo.cpanato.spinmint.com"
, "minioStorageSize":"1Gi"}, "apiVersion":"mattermost.com/v1alpha1", "kind":"ClusterInstallation"}: validation failure list:
spec.databaseType.dbStorageSize in body should match '^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$'
```

Reply:
done @jwilander